server/src/resolvers/getVariantsResolver/utils/fetchCaddAnnotations.ts

```diff
@@ -13,7 +13,7 @@ const ANNOTATION_URL_37 =
 
 const INDEX_37_PATH =
   'https://minio.genomics4rd.ca/www-ssmp-dev/whole_genome_SNVs_inclAnno_GRCh37.tsv.gz.csi';
-const INDEX_38_PATH = '/home/node/cadd_wgs_ghr38_index.gz.tbi';
+const INDEX_38_PATH = 'https://minio.genomics4rd.ca/www-ssmp-dev/cadd_wgs_ghr38_index.gz.tbi';
```

Comment:
```suggestion
const INDEX_38_PATH = 'https://krishna.gs.washington.edu/download/CADD/v1.6/GRCh38/whole_genome_SNVs_inclAnno.tsv.gz.tbi';
```

Reply:
Then I remove the index from the frontend static files?

Reply:
Oh, I didn't realize you uploaded the index to the frontend static files?

When I pasted https://minio.genomics4rd.ca/www-ssmp-dev/cadd_
<img width="860" alt="Screen Shot 2022-04-18 at 2 43 55 PM" src="https://user-images.githubusercontent.com/30064991/163858259-a0bdf7a3-ef58-4580-9786-46552cc09481.png">
wgs_ghr38_index.gz.tbi, got this:

Reply:
No strong preference on my end- just wanna make sure the file is actually there is all!

Reply:
It will be deployed with the frontend when this PR is merged like the `.csi` file. As for which server to use, did you confirm if the hosting lab intended for its downloads to be used as a CDN?

Reply:
Can you elaborate on the CDN point?

Reply:
Does the lab consider directly using the hosted files on its servers by applications acceptable use? Servers and bandwidth don't come for free, so if it's just a download site then it shouldn't be a dependency of this application.

Reply:
I think the collaborators would bias towards using as little resources on the server as possible. The index files are also relatively small (5MB), so I'm not sure how big of a load that is

Reply:
So in conclusion, we should use the remote link that's not in Minio?